library/src/scala/tasty/Reflection.scala

```diff
@@ -41,6 +41,12 @@ class Reflection(val kernel: Kernel)
   val util: reflect.utils.TreeUtils { val reflect: self.type } = new reflect.utils.TreeUtils {
     val reflect: self.type = self
   }
+
+  // TODO remove
+  // For backward compat with macros
+  implicit def reflectionToQuoteContext(implicit reflect: Reflection): scala.quoted.QuoteContext =
+    new scala.quoted.QuoteContext(reflect)
+
```

Comment:
As discussed, in macros it will improve usability if we can reduce the number of concepts.

Reply:
This one will be simplified the next time we update the reference compiler. The _de facto_  implicit will be `QuoteContext`.